draft-ietf-add-resolver-info.md

```diff
@@ -109,9 +105,9 @@ Reputation:
    using the RESINFO RR type and QNAME of "resolver.arpa". In this case, a client has to contend
    with the risk that a resolver does not support RESINFO. The resolver might
    pass the query upstream, and then the client can receive a positive RESINFO response either
-   from a legitimate DNS resolver or an attacker. The DNS client MUST set the Recursion
-   Desired (RD) bit of the query to 0 to ensure that the response is provided by the resolver.
```

Comment:
Shoudn't we keep this part on RD?

Reply:
It is discussed in the WG that setting the RD flag will not solve the problem, see https://mailarchive.ietf.org/arch/msg/add/jZUYZrgqcmTXa6xZlYqNOZS-7wA/